src/main/scala/strawman/collection/View.scala

```diff
@@ -269,6 +253,8 @@ trait IndexedView[+A] extends View[A] with ArrayLike[A] { self =>
     }
   }
 
+  override def knownSize: Int = length
```

Comment:
That’s unrelated to the PR, actually, but I thought we could override `knownSize` in the case of `IndexedView`. We should do the same for `IndexedSeq`.